modules/appeals_api/app/workers/appeals_api/higher_level_review_pdf_submit_job.rb

```diff
@@ -12,23 +11,15 @@ class HigherLevelReviewPdfSubmitJob
     include Sidekiq::Worker
     include CentralMail::Utilities
 
-    def perform(higher_level_review_id, retries = 0)
+    def perform(higher_level_review, retries = 0)
```

Comment:
Nice! So much cleaner!

Reply:
Did we change this to take in an object instead of the id?

Reply:
Yep, removed the ID in favor of the reference. We were doing multiple finds in the job which didn't seem necessary.